docs/name-owners/atomic-name-trading/index.md

```diff
@@ -0,0 +1,294 @@
+
+At the moment, it is possible to make such transfers only using RPC commands to namecoind directly. But in the future, hopefully an easy-to-use UI will be developed that handles all the technicalities. On this page, the basic work-flow will be described with RPC commands. We will assume as an example that Bob wants to buy 'd/my-cool-domain' from Sally for 10 NMC. The work-flow for this transaction is shown below on the test net. 
+
+## Bob Prepares a Buy-Offer
+
+Bob finds the domain of interest and Sally's offer to sell the domain. This could happen, for instance, via Sally using a special value for the domain or by some other channel out of the scope of this page. For instance: 
+
+```
+$ namecoind name_show d/my-cool-domain
+{
+  "name" : "d/my-cool-domain",
+  "value" : "for sale by Sally for 10 NMC, pay to mm1mGfLEMjFPyGsbkcoYiN6yiPcBztK6Dw",
+  "txid" : "00abd8282558574369038722d24c65355e1b98ca3f00d7d09cb7bea8732b598a",
+  "address" : "mt2vTvKfvfb9TKfxB8ns7NVrY4cYzRs73b",
+  "expires_in" : 36000
+}
+```
+
+Now, Bob can prepare an atomic transaction that sends 10 NMC to Sally and transfers __d/my-cool-domain__ at the same time to himself. We assume that Sally wants the 10 NMC sent to __mm1mGfLEMjFPyGsbkcoYiN6yiPcBztK6Dw__, while Bob wants the name to be transferred to __mwKDtLZ9EHSeL8TjkZAcppsnb7M3LRhoGR__. 
+
+To construct the transaction, Bob needs to use the "createrawtransaction" RPC command. It needs three pieces of information: 
+
+* Currency inputs: These will be provided by Bob to pay the 10 NMC to Sally as well as the transaction fee.
+* Currency outputs: 10 NMC to Sally's address and potential change back to Bob.
+* A name_update operation to be performed.
+
+In this example, these three parts will be written in seperate files for simplicity, and then these files will be used in the final RPC command. Of course, it is also possible to write everything directly onto the command-line. 
+
+### Currency Inputs
+
+Assuming a transaction fee of 0.005 NMC is appropriate, Bob needs to find one of his unspent inputs of at least 10.005 NMC to include in the transaction. For this, the "listunspent" command can be used. A suitable input could be this one: 
+
+```
+$ namecoind listunspent
+[
+  {
+    "txid" : "12ed9f83b83e248e2ca19e0197933914013145981fcacc5ba47c27e60eacbfaa",
+    "vout" : 0,
+    "scriptPubKey" : "41040a1b1f8d3b497c7ee13e441539b9e1fe5b4ea96696a4827a74e02aa6c4df4383209de42bbc02ce90e28e5e109b9b1b7dcfd14b02449603c4320faf394719a8f1ac",
+    "amount" : 974.90000000,
+    "confirmations" : 2
+  }
+]
+```
+
+With a size of 974.9 NMC, it is large enough. It will produce __974.9 - 10.005 = 964.895 NMC__ change. The interesting data fields for our purpose are __txid__ and __vout__. They will form the currency input for "createrawtransaction", so Bob writes into the file "tx.in": 
```

Comment:
Also just a tiny nit:  I think that we should use `createrawtransaction` as formatting, not "createrawtransaction".  (And of course the same for other command or variable names, like `txid` and `vout`.)

Reply:
Fixed.